Followup fix for guitest (#333)
owncloud/docs-client-desktop: modules/ROOT/pages/appendices/guitest.adoc

NOTE: If there are problems with starting Squish, please check the log file `~/.squish/squishlibraryx.log.x`. Also make sure that the `~/squish-for-qt-6.x.x/etc/paths.ini` is user-readable.

> ===== Extra Steps for oCIS

Reviewer: ```suggestion
===== Extra Steps for Infinite Scale
```

> While running the tests against oCIS, playwright is used to automate the login process in the browser. Following extra steps need to be taken:

Reviewer: ```suggestion
While running the tests against Infinite Scale, playwright is used to automate the login process in the browser. The following extra steps need to be taken:
```

* xref:run-tests-using-squish[Run Tests Using Squish]
* xref:run-tests-using-docker[Run Tests Using Docker]

> NOTE: Before running middleware, install {yarn-install-url}[yarn] respectively {pnpm-install-url}[pnpm] and clone middleware from {owncloud-test-middleware-url}[here].

Reviewer 2: ```suggestion
NOTE: Before running middleware, install {yarn-install-url}[yarn] and clone middleware from {owncloud-test-middleware-url}[here].
```
